File: translations/codenet/deepseek_commented/C2Go/codellama/Code/s858795471.go
package main

import "fmt"

func search(A []int, n int, key int) int {
    i := 0
    A = append(A, key)
    for A[i] != key {
        i++
    }
    if i != n {
        return 1
    }
    return 0
}

func main() {
    var n, q, key, sum int
    var A []int
    var i int

    fmt.Scan(&n)
    A = make([]int, n)
    for i = 0; i < n; i++ {
        fmt.Scan(&A[i])
    }
    fmt.Scan(&q)
    for i = 0; i < q; i++ {
        fmt.Scan(&key)
        if search(A, n, key) == 1 {
            sum++
        }
    }
    fmt.Println(sum)
}

File: translations/codenet/uncommented/C2P/codellama/Code/s858795471.py
#include<stdio.h>
def search(A,n,key):
    i=0
    A[n]=key
    while A[i]!=key: i+=1
    return i!=n

def main():
    n,A=[],[]
    q,key,sum=0,0,0
    i=0
    
    n=int(input())
    for i in range(n):
        A.append(int(input()))
    q=int(input())
    for i in range(q):
        key=int(input())
        if search(A,n,key): sum+=1
    print(sum)

if __name__ == "__main__":
    main()
